internal/dockerutil/image.go

```diff
@@ -252,11 +260,15 @@ func (c *Container) Wait(ctx context.Context) (stdout, stderr []byte, err error)
 		stderrBuf = new(bytes.Buffer)
 	)
 
-	rc, err := c.image.client.ContainerLogs(ctx, c.containerID, types.ContainerLogsOptions{
+	logOpts := types.ContainerLogsOptions{
 		ShowStdout: true,
 		ShowStderr: true,
-		Tail:       "50",
-	})
+	}
+	if tail != 0 {
+		logOpts.Tail = fmt.Sprint(tail)
```

Comment:
Could be `strconv.FormatUint(tail, 10)` if you want a slightly more focused function.

Reply:
Also (negligible in this case) but strconv will be more performant than `fmt` alternatives.

Reply:
done